src/Note.ts

```diff
@@ -86,6 +93,36 @@ class Note implements PlayableInterface {
 
     return oscillatorNode;
   }
+
+  public getPianoOscillator(audioContext: AudioContext): OscillatorNode {
+    const oscillatorNode = audioContext.createOscillator();
+    const gainNode = audioContext.createGain();
+
+    oscillatorNode.connect(gainNode);
+    gainNode.connect(audioContext.destination);
+
+    const attackTime = 0.01;
+    const decayTime = 0.1;
+    const releaseTime = 0.3;
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** Why these values, are they arbitrary to make a better piano sound?‏